### PR TITLE
runtime: drop qemu-lite support

### PR DESCRIPTION
**Pull request by bergwolf**
backport #2530

### CODE REVIEW

File: docs/how-to/what-is-vm-templating-and-how-do-I-use-it.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ and can be classified as potentially misunderstood behaviors rather than vulnera
 VM templating can be enabled by changing your Kata Containers config file (`/usr/share/defaults/kata-containers/configuration.toml`,
 overridden by `/etc/kata-containers/configuration.toml` if provided) such that:
 
-  - `qemu-lite` is specified in `hypervisor.qemu`->`path` section
+  - `qemu` version `v4.1.0` or above is specified in `hypervisor.qemu`->`path` section
```
Comment (dgibson) [inline docs/how-to/what-is-vm-templating-and-how-do-I-use-it.md:50]: We've already merged changes which require qemu v5.0 or later to work at all, so the specific restriction to qemu v4.1.0 is redundant

**Review by dgibson — APPROVED** [review]

File: docs/how-to/what-is-vm-templating-and-how-do-I-use-it.md (hunk 1 above)
Comment (dgibson) [reply docs/how-to/what-is-vm-templating-and-how-do-I-use-it.md:50]: Oh, sorry, didn't spot that this is a backport branch.  I don't think we do have the v5.0 requirement in stable-2.2, so never mind.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @bergwolf!

**Review by amshinde — APPROVED** [review]

**Pull request merged**